docs/topics/mtr-rn-resolved-issues-1-2-7.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * docs/release-notes-mtr/mtr_release_notes-1.2/master.adoc
+
+:_content-type: REFERENCE
+[id="mtr-rn-resolved-issues-1-2-7_{context}"]
+= Resolved issues
+
+{ProductShortName} 1.2.7 has the following resolved issues:
+
+.MTR 1.2.0 fails with the Exception `java.lang.ClassNotFoundException:org.eclipse.text.edits.MalformedTreeException`
+
+In earlier versions of {ProductShortName} 1.2.z, when migrating an Application from JBoss Enterprise Application Platform (EAP) 7 to EAP 8, there could be a failure with the following `java.lang.ClassNotFoundException`:
+
+[source,java]
+----
+java.lang.ClassNotFoundException: org.eclipse.text.edits.MalformedTreeException from [Module "org.jboss.windup.ast.windup-java-ast:6.3.1.Final-redhat-00002_67e96e90-d3bc-44fe-8fc8-ac2abdeacc58" from AddonModuleLoader]
+----
+
+This issue has been resolved in {ProductShortName} 1.2.7. link:https://issues.redhat.com/browse/WINDUP-4200[(WINDUP-4200)]
+
+.CVE-2022-36033: `org.jsoup/jsoup`: The jsoup cleaner may incorrectly sanitize crafted XSS attempts if SafeList.preserveRelativeLinks is enabled
+
+A flaw was discovered in `jsoup`, which is a Java HTML parser, built for HTML editing, cleaning, scraping, and cross-site scripting (XSS) safety.
+
+An issue in `jsoup` could incorrectly sanitize HTML, including javascript: URL expressions, which could allow XSS attacks when a reader subsequently clicks that link. If the non-default SafeList.preserveRelativeLinks option is enabled, HTML, including javascript: URLs crafted with control characters, will not be sanitized. Users are recommended to upgrade to {ProductShortName} 1.2.7, which resolves this issue.
+
+For more details, see link:https://access.redhat.com/security/cve/2022-36033[(2022-36033)].
```

Comment:
Hi @anarnold97, Could you please check this CVE link? When I previewed it, it redirected to a "Page Not Found" (404 error).

Reply:
@kpunwatk - fixed. thanks

Reply:
It looks fine to me @anarnold97